src/suite.py

```diff
@@ -58,6 +64,7 @@
 FLIP = 1
 DONT_FLIP = 0
 
+NUM_TRIALS = 2

```

Comment:
@CaptainEmerson Can you run the `train_classifier_g` after you change this line to `NUM_TRIALS = 10`?

Reply:
@CaptainEmerson  Also we need results from `convo_word_freq_diff`, but I think we've run it it and the code hasn't changed.. do you still have the result saved somewhere? Or we can just run it again - with the data cleaning you did.

Reply:
I'm running it now, but last time I tried I got:

```
Traceback (most recent call last):
  File "/usr/local/google/home/emersonm/toxicity-detector/bazel-bin/main/train_classifier_g.runfiles/__main__/main/train_classifier_g.py", line 136, in <module>
    trained_model = train_model(training, model_name="rf", what_data=what_data)
  File "/usr/local/google/home/emersonm/toxicity-detector/bazel-bin/main/train_classifier_g.runfiles/__main__/main/train_classifier_g.py", line 77, in train_model
    current_scores = s.self_issue_classification_all(model_name, fid)
  File "/usr/local/google/home/emersonm/toxicity-detector/bazel-bin/main/train_classifier_g.runfiles/__main__/src/suite.py", line 567, in self_issue_classification_all
    N_TRIALS,
NameError: name 'N_TRIALS' is not defined

```

Reply:
Please change N_TRIALS to NUM_TRIALS

Reply:
> @CaptainEmerson Also we need results from convo_word_freq_diff, but I think we've run it it and the code hasn't changed.. do you still have the result saved somewhere? Or we can just run it again - with the data cleaning you did.

What result files do you need? I'm also happy to run it again, but I don't see that I did any data cleaning.

Reply:
I need `train_classifier_16xxxxx.log` and `fighting_words_sorted.csv`. I remember you removed some automatically  generated phrases for the fighting words task, before we showed the results to Carolyn

Reply:
Another dataset is linguistic vs. non-linguistic pushback

Reply:
I need the same files